packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -745,8 +774,8 @@ Future<void> _buildGradleProjectV2(
     command.add('-Pfilesystem-scheme=${buildInfo.fileSystemScheme}');
   if (androidBuildInfo.splitPerAbi)
     command.add('-Psplit-per-abi=true');
-  if (androidBuildInfo.proguard)
-    command.add('-Pproguard=true');
+  if (androidBuildInfo.shrinking)
```

Comment:
nit: maybe `shrink` instead of `shrinking`?

Reply:
Done